### PR TITLE
Cluster labelling UI for adding

### PR DESCRIPTION
**Pull request by ubergesundheit**
Followup to #1474 Implements giantswarm/giantswarm#10541

This PR:

- Increases the granularity of validation messages of cluster keys & values
- Adds a `onClick` prop to the `ValueLabel` component
- Removes the validation error tooltips. Validation errors are now displayed inside the edit tooltip
- Changes editing of labels to allow changes of key and value of a label at the same time
- Move some styles of subcomponents in to own files

This is how it looks (as of 811ab69):
cluster-ui-2

### CODE REVIEW

File: src/components/Cluster/ClusterDetail/ClusterLabels/EditLabelTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    }
+  };
+
+  const keyHandler: KeyboardEventHandler<HTMLInputElement> = ({ key }) => {
+    if (key === 'Escape') {
+      onClose();
+    } else if (
+      keyIsValid === true &&
+      valueIsValid === true &&
+      key === 'Enter'
```
Comment (axbarsan) [inline src/components/Cluster/ClusterDetail/ClusterLabels/EditLabelTooltip.tsx:113]:
```suggestion
      keyIsValid && valueIsValid && key === 'Enter'
```

File: src/components/Cluster/ClusterDetail/ClusterLabels/TempLabel_.tsx_ [hunk 2]
```diff
@@ -0,0 +1,151 @@
+import styled from '@emotion/styled';
```
Comment (axbarsan) [inline src/components/Cluster/ClusterDetail/ClusterLabels/TempLabel_.tsx_:1]: Sup with the name of this file? 😬

File: src/utils/labelUtils.ts [hunk 3]
```diff
@@ -110,14 +110,27 @@ const isQualifiedName: IValidationFunction = (key) => {
       };
   }
 
+  if (name.length === 0) {
+    return {
+      isValid: false,
+      validationError: `Key name part cannot be empty.`,
```
Comment (axbarsan) [inline src/utils/labelUtils.ts:116]:
```suggestion
      validationError: 'Key name part cannot be empty.',
```

File: src/components/Cluster/ClusterDetail/ClusterLabels/TempLabel_.tsx_ (hunk 2 above)
Comment (ubergesundheit) [reply src/components/Cluster/ClusterDetail/ClusterLabels/TempLabel_.tsx_:1]: shouldn't be in the commit :confused:

**Pull request merged**